mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -78,7 +78,8 @@
         Namespace,
         UserFlag,
         ACL,
-        Quota
+        Quota,
+        Attachment
```

Comment:
I wonder is this `Attachment` needed. We already have `Attachment` in `SearchCapabilities` lines below

Reply:
I add attachment to the mailbox capabilities according to my personal understanding from the [jira task description](https://issues.apache.org/jira/browse/JAMES-2156?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20newbie%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC), if it's not required to add, I'll remove them.